<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模型修复</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>修复</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>初始模型结构：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="tB_ms"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>修改后模型结构：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="tB_ms2"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>选择数据集</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pB_upload">
          <property name="text">
           <string>上传文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="tB_uri"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>方式一方式二，多次修复会将之前的修复覆盖</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pB_fix">
        <property name="text">
         <string>修复层</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pB_fix2">
        <property name="text">
         <string>修复通道</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pB_download">
        <property name="text">
         <string>下载模型</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
